gatsby-node.ts

```diff
@@ -370,7 +379,7 @@ export const onCreatePage: GatsbyNode<any, Context>["onCreatePage"] = async ({
   if (isTranslated && hasNoContext) {
     const { isOutdated, isContentEnglish } = await checkIsPageOutdated(
       page.context.intl.originalPath,
-      page.context.language
+      page.context.language!
```

Comment:
What does this do?

Reply:
This? https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

Reply:
Yes, that's true, the `!` asserts that the value is not going to be null or undefined. Useful when you have a definition that is optional or possibly `undefined`, and in the implementation you know for sure that that value exists.

Reply:
However, I think the usage in here is not correct. Will push a new commit fixing this.

Reply:
Made a fix for this, you can see that in the following commit: https://github.com/ethereum/ethereum-org-website/pull/6412/commits/60786027131ffb880facc1ffbf7253ccaeee26fb

The context will always have a language, is not optional anymore. Also, in that commit, I make sure that we are setting the language in all places.